### PR TITLE
NoTrailingWhitespaceInCommentFixer - fix for non-Unix line separators

### PR DESCRIPTION
**Pull request by kubawerlos**
Fixes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4294

I've started with RegEx `/\h+(\R)/`, but this looks cool.

### CODE REVIEW

File: src/Fixer/Comment/NoTrailingWhitespaceInCommentFixer.php [hunk 1]
```diff
@@ -53,14 +53,14 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
     {
         foreach ($tokens as $index => $token) {
             if ($token->isGivenKind(T_DOC_COMMENT)) {
-                $tokens[$index] = new Token([T_DOC_COMMENT, Preg::replace('/[ \t]+$/m', '', $token->getContent())]);
+                $tokens[$index] = new Token([T_DOC_COMMENT, Preg::replace('/(*ANY)[ \t]+$/m', '', $token->getContent())]);
```
Comment (keradus) [inline src/Fixer/Comment/NoTrailingWhitespaceInCommentFixer.php]: `(*ANY)` ? what is it ?

Comment (kubawerlos) [reply src/Fixer/Comment/NoTrailingWhitespaceInCommentFixer.php]: > all Unicode newline sequences

https://www.pcre.org/original/doc/html/pcrepattern.html

**Comment by joelharkes** [comment]
Cool! thanks guys 👍

File: src/Fixer/Comment/NoTrailingWhitespaceInCommentFixer.php (hunk 1 above)
Comment (Slamdunk) [reply src/Fixer/Comment/NoTrailingWhitespaceInCommentFixer.php]: Uau, `Today I Learned`

**Comment by keradus** [comment]
@SpacePossum , I got lost what "PR approved" + "waiting for review" added means

**Comment by keradus** [comment]
Thank you @kubawerlos.

**Comment by SpacePossum** [comment]
> I got lost what "PR approved" + "waiting for review" added means

It means I approve the PR, but it is still needs a second review from another member before it can be merged

**Pull request merged**